### PR TITLE
[11.0][ADD] l10n_nl_account_credit_control: Dutch localization for credit control

### PR DESCRIPTION
**Pull request by tarteo**
This module calculates the dunning fee based on the method defined by the WIK:
<https://www.rijksoverheid.nl/onderwerpen/hulp-bij-schulden/vraag-en-antwoord/hoogte-incassokosten>.

### CODE REVIEW

**Comment by astirpe** [comment]
Some tests of the KvK module are failing now, because the KvK API service changed their response data recently.
I could create a PR for removing the failing assert lines, since they are not decisive for making the tests valid.
I will do it as soon as I have time. Or, If you prefer to speed up your PR, you could do it yourself within this PR, in this case I will approve it quickly.

**Comment by tarteo** [comment]
@astirpe I fixed the tests.

**Review by astirpe — APPROVED** [review]
LGTM

File: l10n_nl_account_credit_control/wizards/credit_control_communication.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                'total_excluded']
+
+            replaces = {
+                '#FEE#': formatLang(
+                    self.env, comm.fee, monetary=True,
+                    currency_obj=comm.currency_id),
+                '#FEENOTAX#': formatLang(
+                    self.env, comm.fee_without_tax, monetary=True,
+                    currency_obj=comm.currency_id),
+                '#FEETAX#': formatLang(self.env, comm.fee_tax, monetary=True,
```
Comment (astirpe) [inline l10n_nl_account_credit_control/wizards/credit_control_communication.py]: Not blocking remark. Could you indent those 2 lines the same way it was done with the previous lines?

File: l10n_nl_account_credit_control/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    'name': 'Credit Control for the Netherlands',
+    'summary': 'Dutch localization for credit control',
+    'license': 'AGPL-3',
+    'author': 'Onestein,Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/l10n-netherlands',
+    'category': 'Localization',
+    'version': '11.0.1.0.0',
+    'development_status': 'Beta',
+    'depends': [
+        'account_credit_control',
```
Comment (hbrunn) [inline l10n_nl_account_credit_control/__manifest__.py:11]: I assume you deliberately don't use https://github.com/OCA/account-financial-tools/tree/11.0/account_credit_control_dunning_fees ?

**Review by hbrunn — COMMENTED** [review]
please rebase

File: l10n_nl_account_credit_control/__manifest__.py (hunk 2 above)
Comment (tarteo) [reply l10n_nl_account_credit_control/__manifest__.py:11]: Yes, correct.

Comment (hbrunn) [reply l10n_nl_account_credit_control/__manifest__.py:11]: still please rebase, then I can merge

Comment (hbrunn) [reply l10n_nl_account_credit_control/__manifest__.py:11]: ...I'm oldfashioned. Just tried out the github editor, seems to work

Comment (tarteo) [reply l10n_nl_account_credit_control/__manifest__.py:11]: I was already trying to do it, but you already solved it thanks :smile:.

**Pull request merged**